Mathmagician.Tests/IntegerTest.cs

```diff
@@ -17,5 +18,19 @@ public void IntegerEnsureICanCreateAnInstance()
             //Assert
             Assert.IsNotNull(my_int);
         }
+        [TestMethod]
+        public void IntegersAddToList()
+        {
+            //Arrange
+            int NumOfIntegersToWrite = 10;
+            Integer intTest = new Integer();
+            List<int> testlist = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+            //Act
+            
+            List<int> actual = intTest.Make_Integers(NumOfIntegersToWrite);
+            //Assert
+            Assert.AreEqual(testlist.Count, NumOfIntegersToWrite);
```

Comment:
This is a good test to determine you get the right number of numbers back, but you'll need to make another test that checks whether it's giving you the correct numbers back.